ChecksFile.wl

```diff
@@ -186,6 +200,9 @@ zero = dispersionCheck[data,\[CapitalOmega],k,En,xflavor]
 VerificationTest[Between[Abs[zero],{-0.01,0.01}],TestID-> "2 Beam \[Omega]\[NotEqual]0 Dispersion Check"]
 
 
+ TestReport["C:\\Users\\Sam\\Documents\\GitHub\\stability_tools\\testfiles.wlt"]

```

Comment:
Use the local path based on the computer plz

Reply:
Will do